{{cookiecutter.repo_name}}/setup.cfg

```diff
@@ -1,10 +1,7 @@
 [isort]
+indent='    '
 line_length = 120
-known_edx =
-known_django = django
-known_djangoapp = model_utils
-known_first_party = {{ cookiecutter.app_name }}
-sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOAPP,EDX,FIRSTPARTY,LOCALFOLDER
```

Comment:
Why doesn't isort need this configuration any more?

Reply:
This was the configuration needed to match the edX Python style guide at the time.  Then there was discussion about that being needlessly complicated and out of sync with what was done in practice in most of the code, so a simpler configuration closer to the isort defaults was picked for edx-platform and the style guide was updated to match.  This just adjusts the cookiecutter to catch up with the changes.

Reply:
Hmm, I wish there were some way devs (like me) could know that these discussions and then decisions are happening.

This section: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order was last changed in December of 2015.  The wiki isn't showing me any other results. Is there another place?

Reply:
Where did you get that date from?  The page was moved to its current location in https://github.com/edx/edx-documentation/pull/1595 in September 2017, Andy altered the import order rules in the previous location via https://github.com/edx/edx-documentation/pull/1511 in June 2017.  The discussion happened at least in https://github.com/edx/cookiecutter-django-app/pull/17 (which I'm going to close now that this version got merged), the HipChat dev room, and in-person conversations (although I doubt if the final decision was covered in all of those places).

Reply:
I see what happened: I used git blame on the three bullets remaining in that section.  Andy's June 2017 change only removed items, so git blame wouldn't tell me when it happened.